markets/aave/commons.ts

```diff
@@ -8,6 +8,7 @@ const MOCK_CHAINLINK_AGGREGATORS_PRICES = {
   BUSD: oneEther.multipliedBy('0.00736484').toFixed(),
   DAI: oneEther.multipliedBy('0.00369068412860').toFixed(),
   ENJ: oneEther.multipliedBy('0.00029560').toFixed(),
+  FEI: oneEther.multipliedBy('0.00042414').toFixed(),
```

Comment:
Please, move this line before the USD mock price please, just to make it easier to follow the evolution of new listings
https://github.com/aave/protocol-v2/pull/200/files#diff-fdb3544e2ccf0395080fe6b84d8bc923ebc899a2b5ac4fb70235d85983ed43c1R28

Reply:
They appear to be alphabetical which is why I put it here. Happy to move it if thats not intentional

Reply:
Whoops, you are right, thanks for the catch!